packages/html/src/colorpicker/tests/colorpicker-states.tsx

```diff
@@ -0,0 +1,53 @@
+import { ColorPickerNormal, ColorPicker } from '../../colorpicker';
+
+const styles = `
+    #test-area {
+        justify-items: start;
+    }
+
+    .k-animation-container {
+        width: min-content;
+        position: relative;
+        overflow: visible;
+    }
+
+    section {
+        display: flex;
+        flex-direction: column;
+    }
+`;
+
+export default () => (
+    <>
+        <style>{styles}</style>
+        <div id="test-area" className="k-d-grid k-grid-cols-4">
+
+            <section>
```

Comment:
Maybe we could demonstrate the different options of the <ColorPickerNormal /> in a separate test -> no color, opacity, icon, icon opacity